libsolidity/interface/OptimiserSettings.h

```diff
@@ -34,9 +34,9 @@ struct OptimiserSettings
 	static char constexpr DefaultYulOptimiserSteps[] =
 		"dhfoDgvulfnTUtnIf"            // None of these can make stack problems worse
 		"["
-			"xarrscLM"                 // Turn into SSA and simplify
+			"xarrscLMS"                // Turn into SSA and simplify
 			"cCTUtTOntnfDIul"          // Perform structural simplification
-			"Lcul"                     // Simplify again
+			"LculS"                    // Simplify again
```

Comment:
Also not exactly sure where to put it.

Reply:
Maybe it should be done together with structural simplification? Do you want to do it twice per cycle?

Reply:
Yeah... I'll just put it at the end of ``structural simplification`` for now...
Until we solve the issue of tracking the "ambient control flow" for all statements (i.e. determine that exiting the switch will always lead to revert, etc.) it's unlikely that this step will really find anything to move anyways I think...